app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -186,6 +193,11 @@
                Can't be greater than {{limits.max | usageWithUnits : 'storage'}}.
              </span>
            </div>
+           <div ng-if="persistentVolumeClaimForm.capacity.$error.underStorage" class="has-error">
+             <span class="help-block">
+               Storage quota will be exceeded - Max allocated: {{storageQuota.maxStorage}} Currently used: {{storageQuota.claimStorageUsed}}.
```

Comment:
this unfortunately can't be this clear cut when throwing cluster quotas into the mix, I think you will need to just link them to the quota page instead of trying to come up with these numbers